packages/create-code/src/index.ts

```diff
@@ -128,6 +128,20 @@ async function run() {
       path.resolve(appDir, "src", "modules")
     );
 
+  // Remove unused pages
+  if (selectedModules.length !== libraryModules.length) {
+    const unusedPages = libraryModules
+      .filter((x) => !selectedModules.includes(x.value))
+      .flatMap(({ pages }) => pages);
+
+    for (const page of unusedPages) {
+      await fse.remove(path.resolve(appDir, "src", "pages", `${page}.tsx`));
+    }
+  }
```

Comment:
The following process is also possible, but it does not seem to be very readable.

```ts
await Promise.all(
    libraryModules
      .filter((x) => !selectedModules.includes(x.value))
      // Get unused page names
      .flatMap((x) => x.pages)
      // Remove unused page files from the `pages` directory
      .map((page) =>
        fse.remove(path.resolve(appDir, "src", "pages", `${page}.tsx`))
      )
  );
```